scripts/ci/job_hooks/after_e2e.sh

```diff
@@ -2,3 +2,5 @@
 
 # Upload protractor-smartrunner artifact related to this particular job to S3
 ./scripts/ci/utils/artifact-to-s3.sh -a "$SMART_RUNNER_DIRECTORY" -o "$S3_DBP_FOLDER/protractor-smartrunner-$TRAVIS_JOB_ID.tar.bz2"
+
+npx @alfresco/adf-cli scan-env --host "$E2E_HOST" -u "$E2E_ADMIN_EMAIL_IDENTITY" -p "$E2E_ADMIN_PASSWORD_IDENTITY"
```

Comment:
in the adf pipeline the cli commands needs to be call from node modules otherwise you are not going to get the new changes